samoa-api/src/main/java/org/apache/samoa/streams/kafka/KafkaJsonMapper.java

```diff
@@ -0,0 +1,103 @@
+/*

```

Comment:
Yes, this is the main difference, as well as the test case where KafkaJsonMapper is being used.